pulse-binding/src/context/introspect.rs

```diff
@@ -258,7 +258,7 @@ impl Drop for Introspector {
 ///
 /// Please note that this structure can be extended as part of evolutionary API updates at any time
 /// in any new release.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Using such a derived clone implementation would just create borrowed copies of the string pointers which seems pointless.